### ISSUE TITLE
dragAndDrop events still run on elements that don't match draggableSelector when their class is changed dynamically

### ISSUE THREAD

**Issue by irrg** [Vivify-Ideas/vue-draggable#23]
I don't have a simplified example as I'm implementing `vue-draggable` into a complex existing solution, but, the following is happening:

Like the example code, I have several ULs with LIs in them. LIs can be toggled to be 'modifying' which is the state in which they can be dragged, renamed, etc. 

```              
<li
  v-if="!modifyStates[item.id]"
  :key="item.file"
  class="file"
  @dragenter="handleDragEnter(category)">
  <div class="label-wrapper">
    <span class="label">{{ item.label }}</span>
  </div>
  <wt-button
    type="secondary"
    icon="pencil-alt"
    size="icon-small"
    @click="setModifyingState(item)">Modify</wt-button>
  <wt-checkbox v-model="openStates[item.id]" />
</li>
<li
  v-else
  :key="item.file"
  class="modifying">
  <div class="label-wrapper">
    <span class="label">DRAG ME</span><!-- (this is a placeholder for more complex functionality) -->
  </div>
  <wt-button
    type="secondary"
    icon="pencil-alt"
    size="icon-small"
    @click="unsetModifyingState(item)">Modify</wt-button>
  <wt-checkbox v-model="openStates[item.id]" />
</li>
```

`draggableSelector` is set to `li.modifying`

After dragging a LI to another UL, and unsetting the `modifying` state, the LI still appears draggable, and events `console.log` (onDrop, onDragstart, onDragend). 

But when dropped, the LI does not move to the other list…this is desireable behavior, so that is good, but it feels like it should no longer be draggable nor should event listeners trigger?

**Comment by nikolasp** [comment]
Hi @irrg,

Thanks for the feedback. I'm working on solution for this. I've made small demo for your use case, so please check out this. It's still work in progress (v2.0.0-beta.0), but next week I hope it will be available stable release.

[![Edit VueDraggable Reactivity](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/8nkk8pw6x0)

**Comment by irrg** [comment]
Demo checks out! Thanks for your work on this.

**Comment by nikolasp** [comment]
Released v2.0.0 🎉
